frontend/src/entry-forms/Transaction.svelte

```diff
@@ -21,12 +21,18 @@
     entry.postings = entry.postings.concat(emptyPosting());
   }
 
+  $: suggestions = $accounts.filter(
+    (value) => $account_details[value].close_date >= new Date(entry.date)
+  );
+
   $: payee = entry.payee;
   $: if (payee) {
-    suggestions = undefined;
+    suggestions = [];
```

Comment:
This is not ideal, since it completely erases the suggestion array during the fetch (or if the fetch fails)